python/hail2/__init__.py

```diff
@@ -1,16 +1,41 @@
 import hail.expr.functions as functions
 import hail.expr.aggregators as agg
 import hail.methods as methods
-from hail.api2 import MatrixTable, Table, HailContext
 from hail.expr import Type, TInt32, TInt64, TFloat32, TFloat64, TString, TBoolean, TArray, TSet, TDict, TStruct, \
     TLocus, TVariant, TAltAllele, TCall, TInterval, eval_expr, eval_expr_typed
 from hail.genetics import LDMatrix, KinshipMatrix, Variant, Locus, AltAllele, Interval, Call, Pedigree, Trio, \
     GenomeReference
 from hail.utils import Struct, hadoop_write, hadoop_read, hadoop_copy
-from hail.api2 import MatrixTable, Table, HailContext, asc, desc
+from hail.api2 import MatrixTable, Table, asc, desc
+from hail.typecheck import nullable, strlike, integral, typecheck
+from hail.utils.java import Env
+from pyspark import SparkContext
 
-__all__ = ['HailContext',
-           'Table',
+def stop():
+    Env.hc().stop()
+
+@typecheck(sc=nullable(SparkContext),
+                  app_name=strlike,
+                  master=nullable(strlike),
+                  local=strlike,
+                  log=strlike,
+                  quiet=bool,
+                  append=bool,
+                  min_block_size=integral,
+                  branching_factor=integral,
+                  tmp_dir=strlike,
+                  default_reference=strlike)
+def init(sc=None, app_name="Hail", master=None, local='local[*]',
```

Comment:
I'm not sure about names as common as stop and init in the top-level scope.  But we're going to switch to `import hail as hl` and `hl.init()` soon?

Reply:
yes. this is a small step to getting there